include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1382,6 +1382,40 @@ struct OrtApi {
                   _In_ const void* model_data, size_t model_data_length,
                   _In_ const OrtSessionOptions* options, _Inout_ OrtPrepackedWeightsContainer* prepacked_weights_container,
                   _Outptr_ OrtSession** out);
+
+  /**
+   * Use this API to create the configuration of a TensorRT Execution Provider.
+   * \param out - pointer to the pointer of TensorRT EP provider options instance.
+   */
+  ORT_API2_STATUS(CreateTensorRTProviderOptions, _Outptr_ OrtTensorRTProviderOptions** out);
```

Comment:
I suggest we change the name of this opaque struct to OrtTensorRTProviderOptionsV2 or a better name so that it doesn't collide with the existing name in the header as this has repercussions on the ReleaseXXX fn.

Reply:
Just to make it clear and correct me if i am wrong.
If we create a new opaque struct called something like OrtTensorRTProviderOptionsV2 for this API, we will need another
SessionOptionsAppendExecutionProvider_TensorRT specific for this new opaque struct, right?
Since currently we only have SessionOptionsAppendExecutionProvider_TensorRT( _In_ OrtSessionOptions* options, _In_ const OrtTensorRTProviderOptions* tensorrt_options) in C API.

Reply:
I've made the modification.